### DISCUSSION TITLE
Running SEB v3.1.1.250 on Windows 11

### DISCUSSION THREAD

**Discussion by guyas** [SafeExamBrowser/seb-win-refactoring#491] [Q&A] [answered]
Hi, any known issues for running SEB_3.1.1.250 on Windows 11?

Our clients have been using it smoothly on Win10 for more than a year. Recently one of them called to say that they were forced to upgrade to Windows 11 and now it just won't work.

Also, is there an option for a corresponding verificator ref file?

EDIT: Project's [FAQ page](https://safeexambrowser.org/faq/faq_en.html) cites an outdated antivirus list. Is there an updated one?

**Comment by dbuechel** [comment, marked as answer]
> Hi, any known issues for running SEB_3.1.1.250 on Windows 11?

Thus far, we are not aware of any. According to our limited internal testing, the latest SEB release seems to work fine on Windows 11.

> [...] they were forced to upgrade to Windows 11 and now it just won't work.

Could you elaborate a bit what exactly doesn't work? Is SEB crashing, are there any warning or error messages or what is happening when your clients try to start SEB?

> Also, is there an option for a corresponding verificator ref file?

Unfortunately not. We highly recommend to always use the latest official release version of SEB for Windows. But you can always create your own reference file with the SEB Verificator and use it to verify an older version of SEB 3.x.

**Reply by guyas** [reply]
ok so there's some more to it. For some unknown reason, clients had this error when running the verificator: 
![Error (2)](https://user-images.githubusercontent.com/8017654/197805889-f4e86111-94bf-485e-84d4-a7db7d0aa744.png)
Weird. It does not require them in win10, and they are nowhere to be found under any installation.

**Reply by dbuechel** [reply]
Well, that would be very strange indeed. The configuration tool is always part of an official installation of SEB. Could you elaborate on what exactly / how exactly you installed and verified SEB?

**Reply by guyas** [reply]
Well, this gets weirder with every day passing.

A former employee of mine has created the program, based on SEB_3.1.1.250. The download link has been distributed to our clients about a year ago and everyone were happy with running it on Win10. So far so good.

When I tried uninstalling and then re-installing today on my win10, everything was ok. Under `C:/Program Files/SafeExamBrowser` I have everything needed for SEB_3.1.1.250 to run properly, and the program runs smoothly.

![Capture5](https://user-images.githubusercontent.com/8017654/197961392-3605dbdf-2265-4d23-953a-f77f06a032c7.PNG)

Under `Application`:

![Capture4](https://user-images.githubusercontent.com/8017654/197961136-2ac4a6f7-b429-4363-8733-5b541edefe73.PNG)

Using the verificator:

![Capture6](https://user-images.githubusercontent.com/8017654/197962580-d2733332-6f19-489e-8778-b0eb2b1f2c9f.PNG)


Then I connected remotely to the client, uninstalled their version and then downloaded using my link (which I personally used minutes earlier). For a reason that is unknown to me, the same installer that's supposed to install version SEB_3.1.1.250, and did do successfully today on my win10 and last week on a win11 of our IT guy - it has installed SEB_3.3.1.388:
![Capture2](https://user-images.githubusercontent.com/8017654/197961126-420b6ccf-574b-44f2-bb17-62d2c34854c1.PNG)
![Capture3](https://user-images.githubusercontent.com/8017654/197961130-c765ecb8-3b9d-46dc-b8bf-4e440d7f3356.PNG)
![Capture1](https://user-images.githubusercontent.com/8017654/197961124-bef03896-0904-4c6c-84a2-5b17683c763f.PNG)


I am lost for words, got no reasonable explanation whatsoever.

**Reply by dbuechel** [reply]
That is most definitely impossible, an installer for a specific SEB version should always install exactly that particular SEB version, we're not doing any bootstrapping or alike. But as you mentioned, you're using a custom build / installer, so there might be something going wrong there.

**Reply by guyas** [reply]
we only wrap it with an NSIS installer, that's it

**Reply by dbuechel** [reply]
It is very hard to judge without knowing the details, but then the error most likely lies there, in the wrapping.

**Reply by guyas** [reply]
OK I just had another remote session with them. It appears that their IT dept have installed SEB_3.3.1.388 before running our installer (why? got no idea, maybe for some other needs). Then, when running our installer, it did practically nothing.

He had to uninstall SEB_3.3.1.388, then run my installer again. After this, it was all working fine.

To sum up: SEB_3.1.1.250 on win11 - excellent; our installer - good; unknown IT workers doing unexplained stuff - not good!

I am very sorry for bothering you with this stupid case. Thank you for both your patient assistance and your great project. Highly appreciated.

**Reply by dbuechel** [reply]
No worries, we know these situations all too well; seemingly random stuff happens to occur all too often 😄
